Stage I/Stage I.py

```diff
@@ -40,16 +53,25 @@
 historical2 = historical2.drop(historical2.columns[4:],axis = 1)
 historical2 = historical2.drop(historical2.columns[0],axis = 1)
 
+# Here I would recomment splitting these operations over multiple lines using
+# the &= operator, so something like
+# a = b & c
+# a &= d
+# a &= e
+# instead of a = b & c & d & e
 pop_dep2 = [historical2.TOT_POP[(historical2.YEAR ==2008) & (historical2.AGE <=19)].sum(),historical2.TOT_POP[(historical2.YEAR ==2009) & (historical2.AGE <=19)].sum()]
 pop_snr2 = [historical2.TOT_POP[(historical2.YEAR ==2008) & (historical2.AGE >=65) & (historical2.AGE < 999)].sum(), historical2.TOT_POP[(historical2.YEAR ==2009) & (historical2.AGE >=65) & (historical2.AGE < 999)].sum()]
 total_pop2 = [historical2.TOT_POP[(historical2.YEAR ==2008) & (historical2.AGE == 999)].sum(), historical2.TOT_POP[(historical2.YEAR ==2009) & (historical2.AGE == 999)].sum() ]
 
 
 #combine the estimates of 08-14 with the projection data
+# the 'df' as an indicator that we are making new DataFrames is distracting here
 POP_DEP = pd.concat([df(pop_dep2),df(pop_dep1),df(pop_projection[pop_projection.columns[1:21]].sum(axis = 1))])
 POP_SNR = pd.concat([df(pop_snr2),df(pop_snr1),df(pop_projection[pop_projection.columns[66:]].sum(axis = 1))])
 TOTAL_POP = pd.concat([df(total_pop2), df(total_pop1.values.transpose()),df(pop_projection.total_pop.values)])
 
+#could also do:
+# Stage_II_targets = DataFrame(TOTAL_POP, columns=['TOTAL_POP'])
```

Comment:
This behavior was not obvious to me, but it turns out that when you call the DataFrame constructor with another DataFrame and a `columns` keyword argument, the `columns` argument is interpreted as 'select these columns from the incoming DataFrame' rather than 'impose these names on the new columns'. So, since there is no 'TOTAL_POP' column in the passed in DataFrame, it creates a new empty column with that name. Bad interface, in my opinion.